projects/igniteui-angular/migrations/update-9_1_0/index.ts

```diff
@@ -4,14 +4,29 @@ import {
     Tree
 } from '@angular-devkit/schematics';
 import { UpdateChanges } from '../common/UpdateChanges';
+import { InputPropertyType } from '../common/schema';
 
 const version = '9.1.0';
 
-export default function(): Rule {
+export default function (): Rule {
     return (host: Tree, context: SchematicContext) => {
         context.logger.info(`Applying migration for Ignite UI for Angular to version ${version}`);
 
         const update = new UpdateChanges(__dirname, host, context);
+        update.addValueTransform('rowSelectable_is_deprecated', function (oldValue: string, bindingType?: InputPropertyType): string {
```

Comment:
Okay, actually looking at this - there's a little bit missing still. For this migration ideally we'd want to convert `[rowSelectable]="true"` to `rowSelection="multiple"` without the extra property binding. You could also land on the case where a old string property `prop="value"` should be updated to an object or something that requires an expression (e.g. `[newProp]="{}"`).

This will require the transform function to be able to return both the value and change the type, possibly best though an object that can also describe the input parameter. This will require more manipulation of the replace group references.